src/index.js

```diff
@@ -440,14 +448,64 @@ export function ConfirmMultisigAddress({
         multisig,
         publicKey,
       });
+    case LEDGER: {
+      // TODO: clean this up. The reason for this is that
+      // we're expecting this malleable object `multisig` that
+      // gets passed in but really these interactions should
+      // just get a braid or something derived from it.
```

Comment:
Not sure if it's appropriate here since this will probably be done soon, but I think it would be good if we began logging TODOs in ClickUp and including a ticket ID in the comment.